.github/workflows/database-tests.yml

```diff
@@ -99,7 +99,7 @@ jobs:
       - name: Coveralls Parallel
         uses: coverallsapp/github-action@v2
         with:
-          flag-name: mssql-node:${{ inputs.node-version }}
+          flag-name: mssql-node:${{ inputs.node-version }}:${{ matrix.mssql-version }}
```

Comment:
I'd get rid the `mssql` tests by client (not database!) version.

Reply:
I renamed this flag so it indicates it's the `mssql-client` version. Did you mean we should drop testing multiple mssql versions entirely, or just stop reporting coverage by client version?

Reply:
I actually did this here some time ago, but currently waiting to drop old node support: https://github.com/typeorm/typeorm/pull/11382/files#diff-1fc7a77e4d6f4244a5d666cf5dbb39e38ff14ef88ca1f0843416af01960e0d34

There is no need anyway to have tests for various client versions since there isn't much of a difference in clients. The difference in behavior is rather between DB versions.

Reply:
Agreed, let's drop it as part of your changes to remove support for old node versions.